Amadey/case_00/idaScript.txt

```diff
@@ -0,0 +1,37 @@
+end = 0x433E59 	# Here is for end of the encoded string's
```

Comment:
Instead using ".txt" file extension for IDAPython script, you can use ".py" extension.